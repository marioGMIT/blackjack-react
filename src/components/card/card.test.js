import React from 'react';
import { shallow } from 'enzyme';

import Card from './card.component'

describe('Card component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Card label="4"/>);
  });

  it('should render CardList component', () => {
    expect(wrapper).toMatchSnapshot();
  });

});