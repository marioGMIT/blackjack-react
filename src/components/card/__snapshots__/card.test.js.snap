// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should render CardList component 1`] = `ShallowWrapper {}`;
